DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditCourseDetails.cshtml

```diff
@@ -0,0 +1,172 @@
+﻿@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@model EditCourseDetailsViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = "Edit course details";
+  var cancelLinkData = new Dictionary<string, string> { { "customisationId", Model.CustomisationId.ToString() } };
+  var customisationId = Model.CustomisationId;
+  const string passwordId = "conditional-password";
+  const string emailId = "conditional-email";
+  const string completionCriteriaId = "conditional-completion-criteria";
+  var passwordCss = "nhsuk-checkboxes__conditional"
+                    + (Model.PasswordProtected ? "" : " nhsuk-checkboxes__conditional--hidden")
+                    + (errorHasOccurred ? " nhsuk-u-padding-left-5" : "");
+  var emailCss = "nhsuk-checkboxes__conditional"
+                 + (Model.ReceiveNotificationEmails ? "" : " nhsuk-checkboxes__conditional--hidden")
+                 + (errorHasOccurred ? " nhsuk-u-padding-left-5" : "");
+  var otherCompletionCriteriaCss = "nhsuk-radios__conditional"
+                                   + (!Model.IsAssessed ? "" : " nhsuk-radios__conditional--hidden")
+                                   + (errorHasOccurred ? " nhsuk-u-padding-left-5" : "");
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(@Model.CustomisationNameSuffix), nameof(@Model.Password), nameof(@Model.NotificationEmails), nameof(@Model.TutCompletionThreshold), nameof(@Model.DiagCompletionThreshold) }" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Edit course details</h1>
+
+    <form class="nhsuk-u-margin-bottom-3" method="post"
+          novalidate
+          asp-controller="ManageCourse"
+          asp-action="SaveCourseDetails"
+          asp-route-customisationId="@customisationId">
+
+      <input type="hidden" asp-for="CentreId" />
+      <input type="hidden" asp-for="ApplicationId" />
+      <input type="hidden" asp-for="CustomisationName" />
+      <input type="hidden" asp-for="PostLearningAssessment" />
+      <input type="hidden" asp-for="DiagAssess" />
+
+      <div class="nhsuk-grid-row nhsuk-u-margin-bottom-5">
+
+        <div class="nhsuk-grid-column-one-quarter nhsuk-heading-s">
+          <label class="nhsuk-heading-s nhsuk-u-font-weight-normal nhsuk-u-margin-bottom-0">
+            Course name
+          </label>
+          <div class="nhsuk-u-margin-top-1">
+            @Model.CustomisationName -
+          </div>

```

Comment:
Do we need nhsuk-heading-s on multiple elements here?

Reply:
removed it from the label, both label and Course name here use the class